Maker-Agents-Suit-Module/OSMAgent/src/big.queued.price.deviation.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  Log,
+  TransactionEvent,
+  Trace,
+  HandleTransaction,
+  Finding,
+  FindingType,
+  FindingSeverity,
+} from "forta-agent";
+import Web3 from "web3";
+
+const PEEK_FUNCTION_SELECTOR = "0x59e02dd7";
+const LOG_VALUE_EVENT_SIGNATURE = "LogValue(bytes32)";
+
+export const createFinding = (contractAddress: string): Finding => {
+  return Finding.fromObject({
+    name: "MakerDAO OSM Contract Big Enqueued Price Deviation",
+    description:
+      "The new enqueued price deviate more than 6% from current price",
+    alertId: "MakerDAO-OSM-1",
+    type: FindingType.Suspicious,
+    severity: FindingSeverity.Info,
+    metadata: {
+      contractAddress: contractAddress,
+    },
+  });
```

Comment:
`everestId` is missing
```suggestion
  return Finding.fromObject({
    name: "MakerDAO OSM Contract Big Enqueued Price Deviation",
    description:
      "The new enqueued price deviate more than 6% from current price",
    alertId: "MakerDAO-OSM-1",
    type: FindingType.Suspicious,
    severity: FindingSeverity.Info,
    everestId: "0xbabb5eed78212ab2db6705e6dfd53e7e5eaca437",
    metadata: {
      contractAddress: contractAddress,
    },
  });
```